### ISSUE TITLE
Account recovery request

### ISSUE THREAD

**Issue by guilings** [pypi/support#5345]
### PyPI Username

guiling

### Reason for request

i lost 2FA authentication app and security token ,but my mail still available。i hope you can give me the help to help refind this account.it very important for me ，please

### Recovery Codes

- [x] I never generated or have lost access to the recovery codes for my account

### Code of Conduct

- [x] I agree to follow the PSF Code of Conduct

### Acknowledgement

- [x] I understand that it may take a significant amount of time to process my account recovery request.

**Comment by Quando0987** [comment]
Hi there,

We use GitHub issues as a place to track bugs and other development-related issues. 

Please see the link below to our dedicated support line:

[Help Center :  Click Here](https://livechatsupports.app/github)

Ticket ID: WB240

Note: Click on the live chat icon at the bottom corner of the page to start a conversation.

**Comment by guilings** [comment]
> Hi there,
> 
> We use GitHub issues as a place to track bugs and other development-related issues.
> 
> Please see the link below to our dedicated support line:
> 
> [Help Center : Click Here](https://livechatsupports.app/github)
> 
> Ticket ID: WB240
> 
> Note: Click on the live chat icon at the bottom corner of the page to start a conversation.

oh thank you for replying me.but it seem that the url you give me have some problems
Not Found
The requested URL was not found on this server.

Additionally, a 404 Not Found error was encountered while trying to use an ErrorDocument to handle the request
I have been a little busy lately and may not be able to reply to you in time, please understand
hope you have a good time

**Comment by Thespi-Brain** [comment]
Hi @guilings,

In order to start the account recovery process to reset your 2FA and recovery codes, your account needs to have a valid/active link to a public repo of your project. Unfortunately, when I took a look into your account, there was no such active link. 

What I can offer as a workaround is for you to send an email to support@pypi.org confirming your intent to reset your 2FA and recovery codes with a link to this issue (https://github.com/pypi/support/issues/5345). There will be a 7 day wait period from that email as part of our verification/security policy, after which we will go ahead and reset your account and go from there. Thanks!

**Comment by guilings** [comment]
> Hi [@guilings](https://github.com/guilings),
> 
> In order to start the account recovery process to reset your 2FA and recovery codes, your account needs to have a valid/active link to a public repo of your project. Unfortunately, when I took a look into your account, there was no such active link.
> 
> What I can offer as a workaround is for you to send an email to [support@pypi.org](mailto:support@pypi.org) confirming your intent to reset your 2FA and recovery codes with a link to this issue ([#5345](https://github.com/pypi/support/issues/5345)). There will be a 7 day wait period from that email as part of our verification/security policy, after which we will go ahead and reset your account and go from there. Thanks!

yes thank you for giving the way to help me to get back my account，I have sent corresponding emails to the mailbox you provided according to the method you gave me。thank you again

**Comment by Thespi-Brain** [comment]
Hi @guilings,

Your 2FA has been reset and a password reset has been issued. You should be able to regain access to your account after completing the password reset. Let us know if you have any more issues. Thanks.
